torch/fx/experimental/symbolic_shapes.py

```diff
@@ -504,23 +514,28 @@ def create_symbolic_sizes_strides_storage_offset(self, ex: torch.Tensor):
                         if stride[i] is None
                     ]
                 )
-                stride[i] = self.create_symbol(val)
+                stride[i] = self.create_symbol(val, sname=f"{sname}.stride({i})")
         assert all(x is not None for x in stride)
         sym_size = [self.create_symintnode(i) for i in size]
         sym_stride = []
-        for stride_expr in stride:
+        for i, stride_expr in enumerate(stride):
             # NB: Don't duck size the stride; instead use the expression
             # we computed
             # TODO: We actually allocated an unnecessary extra symbol
             # here in the smallest unbound stride case, but it's not
             # a big deal because the non-0/1 symbol immediately
             # evaporates from its duck-sizing simplification
-            s = self.create_symbol(val, simplify=False)
-            assert stride_expr is not None
-            assert isinstance(s, sympy.Symbol)
-            self.replacements[s] = stride_expr
+            s: sympy.Symbol
+            if isinstance(stride_expr, SymbolWithSourceName):
+                s = stride_expr
```

Comment:
is this used anywhere? did you mean for `self.replacements[s] = stride_expr` to be tabbed over?

Reply:
oh for             `sym_stride.append(self.create_symintnode(s))`

Tad hard to follow, maybe give the name a little length?